Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -15,6 +16,7 @@ private let KVOLoading = "loading"
 private let KVOEstimatedProgress = "estimatedProgress"
 
 class BrowserViewController: UIViewController {
+    private var urlbar: UrlBar!
```

Comment:
It'd be nice if we could be more consistent about <code>Url</code> vs. <code>URL</code>. It seems like in most places, we use the latter, and so does iOS. So maybe change this to <code>URLBar</code>?